lab-vien/lib/readFiles.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+
+// callback is used in testing
+module.exports = exports = function readFiles(fileArr, callback, writeStream) {
+  var output = [];
+  var count = 0;
+  writeStream = writeStream || process.stdout;
+  fileArr.forEach(function(file) {
```

Comment:
the function passed to forEach has an optional second argument for the current element's index,
you could used that for indexing output array instead of indexOf(file)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach